src/lib/spinel/spinel.h

```diff
@@ -3415,7 +3415,7 @@ enum
      */
     SPINEL_PROP_IPV6_ML_PREFIX = SPINEL_PROP_IPV6__BEGIN + 2,
 
-    /// IPv6 (Unicast) Address Table
+    /// IPv6 (Unicast) Address Table - Deprecated
```

Comment:
nit: How about adding more description: ` - Deprecated. This property is replaced by SPINEL_PROP_IPV6_UNICAST_ADDRESS_TABLE`.

Reply:
Updated! Added the statement in below comments.